.eslintrc.js

```diff
@@ -16,5 +16,6 @@ module.exports = {
     'ecmaVersion': 2018,
   },
   'rules': {
+    'max-len': [2, 88, 4]
```

Comment:
What do these numbers mean?

Reply:
Hey, so this was to overide the default linting rule about line length. The first number decides the type of error, either 0,1,2. 0 means no rule, 1 means warning, 2 means error if I remember correctly. 88 is the numbers of characters allowed per line. 4 in the amount of indentation I believe, although this might need to be 2? Not sure

Reply:
Thanks!